DOCUMENTATION.md

```diff
@@ -55,17 +64,14 @@ If you are referencing this package from a CI tool you may need to add the bintr
   </packageSources>
 </configuration>
 ```
+_Is this CI tool specific? is there any extra supporting info that is needed?_
 
-</details>
-
-### [Visual Studio](https://www.visualstudio.com/) (Windows)
+### Visual Studio for Windows
 
-To execute the NUnit tests you will need to install the [NUnit3 Test Adapter](https://marketplace.visualstudio.com/items?itemName=NUnitDevelopers.NUnit3TestAdapter) extension to Visual Studio or via the [NUnit3TestAdapter](https://www.nuget.org/packages/NUnit3TestAdapter/) nuget package
+#### Set the Windows environment variables for Visual Studio for Windows
```

Comment:
I'm not sure why the environment variables are mentioned. They are only needed when running the integration tests and not for using the notification-net-client

Reply:
They could be in the contribution.md but not here.